docs/guides/admin/docs/workflowoperationhandlers/process-smil-woh.md

```diff
@@ -0,0 +1,152 @@
+
+The SMIL file can use more than one source video, but the caller has to take care that the dimension of
+all the source videos are the same.
+This workflow will generate one independent ffmpeg operation per SMIL paramgroup (based on source) regardless
+of the number of target outputs.
+
+This workflow can handle each source flavor selector independently.
+eg: Each source selector can have its own set of encoding profiles, target tags and flavors.
+The parameters for each configuration, such as flavor are separated into sections by "**;**".
+E
+ach source media selector can have its own sets of encoding profile ids (one for each target recording)
+and target tags,
+as well as its own set of target tags and flavors, defined as a comma delimited list.
+
+
+As an example, using presenter/source and presentation/source as uploaded media.
+eg:
+```
+ <configuration key="source-flavors">*/source</configuration>
+```
+>One source selector means that all the matching recording will be processed the same way.
+
+```
+ <configuration key="source-flavors">presenter/source;presentation/source</configuration>
+```
+>Two different source selectors separated by semicolons means that all the matching recordings in the
+>first selector will be processed according to the parameters in the first
+>section and the all the
+>matching recordings in the second selector will be processed according to the parameters in next section
+>of the other configuration values such as encoding profiles.
+
+Each source selector can have only one corresponding section in each set of values.
+The use of the semi-colon is optional. If it is absent, there is only one section.
+If there is only one source selector, but multiple sections in the parameters, then the sections are collapsed
+into one and they will apply to all the source flavors in the source selector.
+"N to N" means that each section has its own processing configuration.
+"1 to N" or "N to 1" means that all the sections are processed the same way,
+ but "M to N" where "M <> N" will result in an error.
+
+eg:
+```
+<configuration key="target-flavors">*/preview</configuration>
+<configuration key="encoding-profiles">mp4-low.http;mp4-vga-medium</configuration>
+```
+>All targets are flavored the same way.
+>Using the example above,
+>all media are encoded with "mp4-low.http" and "mp4-vga-medium" and
+>targets are flavored as "presenter/preview" and "presentation/preview"
+
+```
+ <configuration key="target-tags">engage-streaming,rss,atom;engage-download,rss,atom</configuration>
+ <configuration key="encoding-profiles">mp4-medium.http;mp4-vga-medium</configuration>
+```
+
+>Each section is tagged individually. Using the example above,
+>presenter/preview is encoded with "mp4-medium.http" and tagged with "engage-streaming" ,"rss" and "atom",
+>presentation/preview is encoded with "mp4-vga-medium" and tagged with "engage-download","rss" and "atom".
+
+
+## Example:
```

Comment:
Remove trailing colon in title

Reply:
Example
Parameter Table
Operation Example

could be collapsed to

Parameter Table
Operation Example (incl. Example(